templates/web/oxfordshire/report-video.html

```diff
@@ -0,0 +1,9 @@
+<div class="video-thumbnail-section">
+    <a class="iframe fancybox" rel="fancy" href="https://www.youtube.com/embed/2mxbgJXGHTQ">
+        <img class="video-thumbnail" alt="What we can and can’t fix video" src="/cobrands/oxfordshire/images/oxfordshire-can-cants-video.png">
+        <div class="video-cap-wrapper">
+            <p style="color:#222"><strong>Watch</strong></p>
+            <p>Cllr Tim Bearder explains what we can and can’t fix</p>
+        </div>
+    </a>
+</div>
```

Comment:
This template doesn't seem to be included anywhere?

Reply:
Thanks for the feedback @chrismytton. About this I'm not sure where to include the conditional that test IF Oxfordshire then include the video. 
At the beginning I was going to include it with Jquery, but with Moray we thought would be better to include in a template and then add a conditional to test if the council is Oxfordshire.

Reply:
I think if you move report-video.html to be report/new/top_message.html does that then appear where you would like? That is a cobrand-specific template for a top message.

Reply:
@dracos yes that's perfect, Matthew =)